dockerGrader/testServer.py

```diff
@@ -72,6 +81,32 @@ def fetchFromS3(project, netId):
     curPath = ACCESS_PATH.format(project=project, googleId=googleId)
     downloadSubmission(curPath)
 
+def containerStatus(dockerId):
```

Comment:
containerId might be more specific (because images also have IDs, and we aren't referring to those)